src/gsy_e/models/area/__init__.py

```diff
@@ -221,22 +235,25 @@ def area_reconfigure_event(self, **kwargs):
                             export_capacity_kVA=export_capacity_kVA
                         )
 
-        except Exception as ex:
+        except (GSyAreaException, GSyDeviceException) as ex:
             log.error(ex)
-            return
+            return None
 
         self._set_grid_fees(grid_fee_constant, grid_fee_percentage)
         self.throughput = throughput
-        self._update_descendants_strategy_prices()
+        self.update_descendants_strategy_prices()
+
+        return None
 
-    def _update_descendants_strategy_prices(self):
+    def update_descendants_strategy_prices(self):
```

Comment:
Why changing this method to public ? From a quick search this is not called from anywhere else in the code, therefore it makes sense to stay private.

Reply:
That's okay then, it makes sense to be public in this case! Thanks for explaining!